src/panels/config/network/supervisor-network.ts

```diff
@@ -488,7 +485,8 @@ export class HassioNetwork extends LitElement {
   }
 
   private _handleRadioValueChanged(ev: CustomEvent): void {
```

Comment:
```suggestion
  private _handleRadioValueChanged(ev: Event): void {
```
This is a change event, not a custom event